[2024-09-12T02:23:02.462+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:23:02.552+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V6.second_task scheduled__2023-07-10T00:00:00+00:00 [queued]>
[2024-09-12T02:23:02.596+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V6.second_task scheduled__2023-07-10T00:00:00+00:00 [queued]>
[2024-09-12T02:23:02.598+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:23:02.647+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2023-07-10 00:00:00+00:00
[2024-09-12T02:23:02.670+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9838) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:23:02.674+0000] {standard_task_runner.py:72} INFO - Started process 9868 to run task
[2024-09-12T02:23:02.673+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V6', 'second_task', 'scheduled__2023-07-10T00:00:00+00:00', '--job-id', '20578', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpp7lg_lv4']
[2024-09-12T02:23:02.680+0000] {standard_task_runner.py:105} INFO - Job 20578: Subtask second_task
[2024-09-12T02:23:02.905+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V6.second_task scheduled__2023-07-10T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:23:03.352+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='acap' AIRFLOW_CTX_DAG_ID='our_first_dag_V6' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-07-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-10T00:00:00+00:00'
[2024-09-12T02:23:03.356+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:23:03.403+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:23:03.405+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the second task!']
[2024-09-12T02:23:03.423+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:23:03.426+0000] {subprocess.py:93} INFO - hey, this is the second task!
[2024-09-12T02:23:03.427+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:23:03.482+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:23:03.484+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V6, task_id=second_task, run_id=scheduled__2023-07-10T00:00:00+00:00, execution_date=20230710T000000, start_date=20240912T022302, end_date=20240912T022303
[2024-09-12T02:23:03.598+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:23:03.626+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-12T02:23:03.638+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
